qa/suites/ceph-disk/basic/tasks/ceph-detect-init.yaml

```diff
@@ -0,0 +1,11 @@
+openstack:
+- volumes: # attached to each instance
+    count: 0
+roles:
+- [client.0]
+tasks:
+- install:
+- exec:
+    client.0:
+      - ceph-detect-init
```

Comment:
why do we want to run it twice?

Reply:
Allow me to turn the question around: why would we *not* want to run it twice? :-P

Seriously, I want to run it twice because I like the visual feedback of seeing this in the teuthology log:

```
+ ceph-detect-init
systemd
```